src/extension.ts

```diff
@@ -219,10 +219,9 @@ export async function activate(context: vscode.ExtensionContext) {
         await loginWithMostRecentCredentials(toolkitSettings, loginManager)
 
         recordToolkitInitialization(activationStartedOn, getLogger())
-    } catch (error) {
-        const channelLogger = getChannelLogger(toolkitOutputChannel)
-        channelLogger.error('AWS.channel.aws.toolkit.activation.error', 'Error Activating AWS Toolkit', error as Error)
-        throw error
+    } catch (e) {
+        getLogger().error(localize('AWS.channel.aws.toolkit.activation.error', 'Error Activating AWS Toolkit'), e)
```

Comment:
Don't localize a log-only error message. Maybe stick to the channel logger since this will also write to the main output channel (not just the log one)

Reply:
This was already this way, but reverted the change to `getLogger`